Kube/scripts/deploy-shield.sh

```diff
@@ -83,12 +84,15 @@ get_timezone() {
 
     echo $TZ
 }
-
+FIRST_NAMESPACE="true"
 only_namespace() {
-    SHIELD_MNG="no"
-    SHIELD_PROXY="no"
-    SHIELD_FARM="no"
-    SHIELD_ELK="no"
+    if [ $FIRST_NAMESPACE == "true" ]; then
```

Comment:
check if we already entered this function before..
if not then reset all namespaces deploy to no.